Config.cs

```diff
@@ -1,44 +1,49 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using Newtonsoft.Json;
+using System;
+using CitizenFX.Core;
+using CitizenFX.Core.Native;
+using System.IO;
 
 namespace FRFuel
 {
-    public class Config
+    class Config
     {
-        protected Dictionary<string, string> Entries { get; set; }
+        private static Config _instance;
```

Comment:
I don't see any reason why it needs to be a singleton, instead, it could still be fine being one instance in `FRFuel` class object anyway.

Reply:
One instance of an object is the same as a singleton.